sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -253,6 +253,39 @@ class CudfMin(ref: Expression) extends CudfAggregate(ref) {
   override def toString(): String = "CudfMin"
 }
 
+class CudfTDigest(
+    ref: Expression,
+    percentileExpr: GpuLiteral,
+    accuracyExpression: GpuLiteral)
+  extends CudfAggregate(ref) {
+
+  // map Spark delta to cuDF delta - this is experimental for now and
+  // will not guarantee that we have better results than Spark although
+  // it did help with one of the tests
+  private lazy val accuracy = accuracyExpression.value.toString.toInt match {
```

Comment:
We know the type of value is always an int so just cast it.

`accuracyExpression.value.asInstanceOf[Int]`. Or better yet you already are doing a match. Just combine it.

``` 
private lazy val accuracy = accuracyExpression.value match {
    case delta: Int if delta >= 10000 => 10000
    ...
}
```